addons/cookoff/XEH_postInit.sqf

```diff
@@ -81,6 +81,7 @@ GVAR(cacheTankDuplicates) = call CBA_fnc_createNamespace;
 ["AllVehicles", "killed", {
     params ["_vehicle"];
     if (_vehicle getVariable [QGVAR(enableAmmoCookoff), GVAR(enableAmmoCookoff)]) then {
+        if (GVAR(ammoCookoffDuration) == 0) exitWith {};
```

Comment:
I almost feel like we should just replace the `enableAmmoCookoff` setting with this new duration setting (can just be set to 0 for disabled ammo cook off and provides more control).